src/module/actor/hazard/sheet.ts

```diff
@@ -101,15 +102,15 @@ export class HazardSheetPF2e extends ActorSheetPF2e<HazardPF2e> {
         };
     }
 
-    private prepareActions(): HazardActionSheetData {
+    #prepareActions(): HazardActionSheetData {
         const actions = this.actor.itemTypes.action.sort((a, b) => a.sort - b.sort);
         return {
             reaction: actions.filter((a) => a.actionCost?.type === "reaction"),
             action: actions.filter((a) => a.actionCost?.type !== "reaction"),
         };
     }
 
-    private prepareSaves(): HazardSaveSheetData[] {
+    #prepareSaves(): HazardSaveSheetData[] {
```

Comment:
I thought we couldn't do such things with document classes? Does this work in a production build?

Reply:
This is a sheet class